### DISCUSSION TITLE
检测到您的页面展示可能收到浏览器插件影响巴拉巴拉, 请问咋解决这个问题

### DISCUSSION THREAD

**Discussion by LucasIsGood** [the1812/Bilibili-Evolved#4171] [提问 / Q&A]
![Snipaste_2023-04-30_17-17-55](https://user-images.githubusercontent.com/56297860/235345698-d76ee6b7-f970-422b-9b0a-1657f04a94fd.png)
检测到您的页面展示可能收到浏览器插件影响巴拉巴拉, 请问咋解决这个问题, 在mac的safari就没有这个问题, 在win上就有..

**Comment by Geeyun-JY3** [comment]
#3719
